docs/src/index.md

```diff
@@ -336,15 +372,15 @@ attr = a_open(parent, name)
 t = t_open(parent, name)
```

Comment:
```suggestion
t = open_datatype(parent, name)
```
(and similar for previous three lines. For some reason I can't get Github to let me add a suggestion for those lines...)